app/forms/additional_claimants_form.rb

```diff
@@ -36,8 +36,8 @@ class ClaimantForm < Form
     attribute :has_representative, :boolean
 
 
-    validates :date_of_birth, date: true, presence: true, comparison: {less_than: Date.today}
-
+    validates :date_of_birth, date: { in_the_past: true }, presence: true
+    validate :date_is_past
```

Comment:
I don't understand why we have the date validator checking that the date is in the past AND a method called date_is_past as well doing the same thing ?